manual/moderncv_userguide.tex

```diff
@@ -729,6 +777,13 @@ \subsubsection{Icons}
   \renewcommand*{\fixedphonesymbol}{\ding{38}~}
 \end{lstlisting}
 
+\paragraph{Alternative to icons}
+In case \texttt{symbols} (default) are not wished there is also the option to use \texttt{letters} instead. This is defined by the macro \code{\\moderncvicons}. It must be called after setting the style in the preamble.
```

Comment:
```suggestion
In case you do not want \texttt{symbols} (default) for the icons, there is also the option to replace icons with \texttt{letters} instead. This is set by the macro \code{\\moderncvicons}, which must be called after setting the style in the preamble.
```

Reply:
Corrected with [303fd96](https://github.com/moderncv/moderncv/pull/184/commits/303fd9693cde6846ac9b79cf77368aba58252ed0)